examples/jsm/renderers/webgpu/WebGPURenderer.js

```diff
@@ -71,6 +72,9 @@ class WebGPURenderer {
 
 		this._renderTarget = null;
 
+		this._antialias = !! parameters.antialias;
+		this._sampleCount = this._antialias ? 4 : 1; // Should be configurable?
```

Comment:
I vote to make this configurable with a new parameter `sampleCount`.

Reply:
That sounds good to me. Which do you prefer?

1. renderer constructor takes `antialias` and `sampleCount`. `sampleCount` is effective only `antialias` is `true`. Default `sampleCount` is 4. Pros is maybe `antialias` option would be intuitive especially to existing users.

2. renderer constructor takes only `sampleCount`. That it is greater than 1 means antialiasing is enabled. Pros is less the number of option parameters would be good.

Reply:
I vote for option 1.

Reply:
Thanks, updated.